### PR TITLE
Fix issue #176

### PR DESCRIPTION
**Pull request by sav007**
@krisorr @yervantk

### CODE REVIEW

File: MobileBuy/sample/src/main/java/com/shopify/sample/activity/ProductListActivity.java [hunk 1]
```diff
@@ -71,6 +71,9 @@ protected void onCreate(Bundle savedInstanceState) {
         Intent intent = getIntent();
         if (intent.hasExtra(EXTRA_COLLECTION_ID)) {
```
Comment (krisorr) [inline MobileBuy/sample/src/main/java/com/shopify/sample/activity/ProductListActivity.java]: could we do:
```
collectionId = null;
if (intent.hasExtra(EXTRA_COLLECTION_ID)) {
    collectionId = intent.getLongExtra(EXTRA_COLLECTION_ID);
}

File: MobileBuy/sample/src/main/java/com/shopify/sample/ui/ProductDetailsFragment.java [hunk 2]
```diff
@@ -206,6 +206,9 @@ private void initializeBuyClient() {
 
         // Retrieve the id of the Product we are going to display
         productId = bundle.getLong(ProductDetailsConfig.EXTRA_SHOP_PRODUCT_ID, -1);
```
Comment (krisorr) [inline MobileBuy/sample/src/main/java/com/shopify/sample/ui/ProductDetailsFragment.java]: lets be consistent between here and the location where we get the collectionId

Comment (sav007) [inline MobileBuy/sample/src/main/java/com/shopify/sample/ui/ProductDetailsFragment.java]: ?

Comment (sav007) [inline MobileBuy/sample/src/main/java/com/shopify/sample/ui/ProductDetailsFragment.java]: Agh got it

File: MobileBuy/sample/src/main/java/com/shopify/sample/activity/ProductListActivity.java (hunk 1 above)
Comment (sav007) [inline MobileBuy/sample/src/main/java/com/shopify/sample/activity/ProductListActivity.java]: We don't even need collectionId = null; as it always null

**Comment by krisorr** [comment]
LGTM

**Pull request merged**